src/common/snippets/src/pass/collapse_subgraph.cpp

```diff
@@ -9,6 +9,7 @@
 #include "snippets/op/subgraph.hpp"
```

Comment:
just random place: could you please add these ops [here](https://github.com/openvinotoolkit/openvino/blob/master/src/common/snippets/include/snippets/snippets_isa_tbl.hpp)?

Reply:
done, thanks. But full type names (with namespaces) are used in snippets, looks like it's not used.